src/main/java/org/jenkinsci/plugins/pipeline/utility/steps/conf/AbstractPropertiesStep.java

```diff
@@ -0,0 +1,19 @@
+package org.jenkinsci.plugins.pipeline.utility.steps.conf;
+
+import org.jenkinsci.plugins.workflow.steps.Step;
+
+import java.util.Map;
+
+public abstract class AbstractPropertiesStep extends Step {
+    public abstract String getFile();
+
+    public abstract void setFile(String file);
```

Comment:
why having these 'abstract'? Would be less code duplication in the other classes if the fields and annotations where in this class instead.